assets/sass/05-blocks/tag-clould/_style.scss

```diff
@@ -0,0 +1,4 @@
+.wp-block-tag-cloud {
+	padding-left: var(--global--spacing-unit);
+	padding-right: var(--global--spacing-unit);
```

Comment:
These paddings should only apply to `alignfull`, otherwise it adds extra space in the default view:

<img width="663" alt="Screen Shot 2020-10-22 at 10 53 52 AM" src="https://user-images.githubusercontent.com/541093/96889746-e6fbc480-1454-11eb-9bc5-61dc9146fdfa.png">

Reply:
Should be fine now.